### ISSUE TITLE
2 slides - strange animation

### ISSUE THREAD

**Issue by admench** [gilbitron/Ideal-Image-Slider#82]
Hi ,

When only 2 slides are present, the animation is strange... Adding a 3rd slide fixes this. Any tips?

![jun-15-2016 09-05-23](https://cloud.githubusercontent.com/assets/672916/16072362/5cfc252a-32d8-11e6-9936-d95e197a4a4c.gif)

**Comment by b3wii** [comment]
Yes i also noticed this. Its sad because this slider is very nice. Please fix this.
